### ISSUE TITLE
Remove erroneous folder google/monitoring

### ISSUE THREAD

**Issue by parthea** [googleapis/python-monitoring-dashboards#73]
We have an erroneous google/monitoring folder rather than google/cloud/monitoring. This folder should be removed to avoid confusion. 
https://github.com/googleapis/python-monitoring-dashboards/tree/master/google/monitoring

**Comment by danoscarmike** [comment]
@busunkim96 is there any reason why this couldn't be just deleted? It seems like the correct directory structure (`google/cloud/monitoring_dashboard*`) is present and being updated.

**Comment by busunkim96** [comment]
@danoscarmike My only concern is that if the incorrect path is usable removing it would technically be a breaking change. I'll check in a bit and update this issue.

**Comment by danoscarmike** [comment]
@busunkim96 you're absolutely right. I just checked and it is possible to import via the erroneous path.

In a clean virtual env the following works without error:

```bash
pip install --upgrade google-cloud-monitoring-dashboards
```
```python
from google.monitoring import dashboard
client = dashboard.DashboardsServiceClient()
```

**Comment by parthea** [comment]
I'm going to transfer this issue to google-cloud-python as we're planning to move the code for ` google-cloud-monitoring-dashboards` there in the next 1-2 weeks
